app/utils/callback/lava.py

```diff
@@ -640,6 +642,39 @@ def _add_rootfs_info(group, base_path, file_name="build_info.json"):
         utils.LOG.warn("ValueError: {}".format(e))
 
 
+def _adjust_log_line_numbers(results, log):
+    """ A workaround for a race condition effect visible in LAVA callbacks
+
+    This function changes log end line numbers in LAVA test results to match
+    the moment when the signal was sent instead of when it was received.
+    Function doesn't return value, but modifies results dictionary.
+    """
+    for ts, result_yaml in results.items():
+        if ts == 'lava':
+            continue
+        results_data = yaml.load(result_yaml, Loader=yaml.CLoader)
+        for result in results_data:
+            end_line_number = int(result['log_end_line'])
+            log_end_line = log[end_line_number]
+            test_case_rcvd = SIGNAL_RECEIVED_PATTERN.match(log_end_line['msg'])
+            test_case_id = test_case_rcvd.group(1)
```

Comment:
It's causing exceptions in the backend log:
```
Jul 02 21:43:44 kernelci-staging kernelci-celery[3244]:     test_case_id = test_case_rcvd.group(1)
Jul 02 21:43:44 kernelci-staging kernelci-celery[3244]: AttributeError: 'NoneType' object has no attribute 'group'
```

Reply:
Added  `None` handling when `re.match` didn't match any substrings.